cmd/operator-sdk/add/api.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 The Operator-SDK Authors
+// Copyright 2020r The Operator-SDK Authors
```

Comment:
typro

Reply:
We only use `2020` for files initially created in 2020. Existing files should not change.

Reply:
Really? heh we used to update them every year on my other projects. On RHN I think we actually had a range 2004-current year. Thanks for the info.